src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -168,6 +171,15 @@ public AddTravelPlanCommand parseTravelPlan(String args) throws ParseException {
         WanderlustDate startDate = ParserUtil.parseDate(argMultimap.getValue(PREFIX_START).get());
         WanderlustDate endDate = ParserUtil.parseDate(argMultimap.getValue(PREFIX_END).get());
 
+        if (!TravelPlan.isValidStartAndEndDate(startDate, endDate)) {
+            throw new ParseException(MESSAGE_INVALID_STARTANDENDDATE);
+        }
+
+        if (!TravelPlan.isStartDateAfterToday(startDate)) {
```

Comment:
Same as above. Or is it becos of the error message?

Reply:
Different error message